src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -280,9 +280,15 @@ private static void printPropertyValue(String key, String value) {
     /*
      * prints the locale subopt/section
      */
-    private static void printLocale() {
+    private static void printLocale(boolean summaryMode) {
         Locale locale = Locale.getDefault();
-        ostream.println(LOCALE_SETTINGS);
+        if (!summaryMode) {
+            ostream.println(LOCALE_SETTINGS);
+        } else {
+            ostream.println("Locale settings summary:");
+            ostream.println(INDENT + "Use \":locale\" " +
```

Comment:
Hello Sean, should this instead say "Use \"-XshowSettings:locale\" ...." so that it's clear that this is a sub-option for the -XshowSettings option?

Reply:
Yes, no harm to be clearer Jai. Done.